getting_started/cavs-boot/apollolake/apl-boot-ldr.rst

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
It's very handy to add a unique document label at the top of each file.  This gives you the ability to use Sphinx's cross reference linking using ``:ref:`label_name` `` to create links between documents.  You can also drop a (unique) label within documents too to make it easier to link to specific sections of another document.

Syntax is:

    .. _apl boot loader:

    Apollolake Boot Loader
    ######################

(Note the leading underscore on that label.) Then in another document you can write:

    Refer to the :ref:`apl boot loader` documentation for further information.